ext/MitsubaLoader/CMitsubaMaterialCompilerFrontend.cpp

```diff
@@ -244,44 +244,30 @@ namespace MitsubaLoader
         case CElementBSDF::THINDIELECTRIC:
         case CElementBSDF::ROUGHDIELECTRIC:
         {
-            nextSym = ir->allocNode<IR::CBSDFCombinerNode>(IR::CBSDFCombinerNode::ET_FRESNEL_BLEND);
-            nextSym->children.count = 2u;
+            auto* dielectric = ir->allocNode<IR::CDielectricBSDFNode>();
+            nextSym = dielectric;
 
             const float eta = current->dielectric.intIOR/current->dielectric.extIOR;
```

Comment:
this needs to be the other way round if you're constructing the IR AST for the backface

Reply:
thats how it's done ;)